kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -659,21 +657,25 @@
         option == 'search' ? (this.mobileSearchActive = true) : (this.mobileSearchActive = false);
         this.sidePanelIsOpen = !this.sidePanelIsOpen;
       },
+      // Stick the side panel to top. That can be on the very top of the viewport
+      // or right under the 'Browse channel' toolbar, depending on whether the toolbar
+      // is visible or no (the toolbar hides on smaller resolutions when scrolling
+      // down and appears again when scrolling up).
+      // Takes effect only when the side panel is not displayed full-screen.
       stickyCalculation() {
-        let header = document.getElementsByClassName('header')[0];
-        let topbar = document.getElementsByClassName('ui-toolbar')[0];
-        if (header) {
-          let position = header.getBoundingClientRect();
-          let topbarPosition = topbar.getBoundingClientRect();
-          if (position.bottom >= 64) {
-            this.stickyTop = `${position.bottom}px`;
-          } else if (position.bottom < 0 && topbarPosition.bottom < 0) {
-            this.stickyTop = '0px';
-          } else {
-            this.stickyTop = '64px';
-          }
+        const header = this.$refs.header;
```

Comment:
`let header = document.getElementsByClassName('header')[0]` seems like a riskier reference, as it could pick up a `header` class from anywhere in the DOM, whereas this will get the header specifically in this component.

Reply:
(also `let` is unnecessary unless we are intending to reassign the `header` variable later)

Reply:
Yes, I made those changes to make things a bit safer in that regards.